public/static/docs/api-reference/get_url.md

```diff
@@ -82,10 +83,13 @@ resource_url = dvc.api.get_url(
     'get-started/data.xml',
     repo='https://github.com/iterative/dataset-registry'
     )
+
 print(resource_url)
```

Comment:
Kind of like it better without the extra new line 😋

Reply:
it just makes it easier to read this way ... probably no need in saving lines :) up to you though

Reply:
Too minor. The option would be to wrap in `print()` (esp. if we do the same in the basic usage section on top). But resolving this...